### ISSUE TITLE
Bulk edit functions

### ISSUE THREAD

**Issue by xkl** [TandoorRecipes/recipes#1041]
**Is your feature request related to a problem? Please describe.**
After a failed import I got a lot of bad recipes and deleting them is a tedious task for each recipe: edit, delete, confirm.

**Describe the solution you'd like**
A list view of all recipes with filters and sorting where I can select a bunch of recipes and perform tasks batch-wise.

**Comment by vabene1111** [comment]
Yes that would definitely be something good. Maybe even selecting based on tags, creation time, name filters or similar.

**Comment by TWART016** [comment]
That would be great. Also also add multiple recipes to books/shoppinglist/log/print/export/share.
